### PR TITLE
Add Apple Hypervisor (applehv) to default artifacts

### PR DESCRIPTION
**Pull request by baude**
See coreos/fedora-coreos-tracker#1533 and coreos/fedora-coreos-tracker#1548

### CODE REVIEW

**Comment by dustymabe** [comment]
Converting to draft. IIUC this requires some work to land in Ignition and possibly a few other places (afterburn, fedora-coreos-config) before we want to start generating artifacts.

**Comment by dustymabe** [comment]
The actual changes here look good and we should be able to merge them when all the other deps are in place.

File: config.yaml [hunk 1]
```diff
@@ -50,6 +50,7 @@ default_artifacts:
     - openstack
     - hashlist-experimental
   aarch64:
+    -applehv
```
Comment (dustymabe) [inline config.yaml:53]:
```suggestion
    - applehv
```

**Review by dustymabe — CHANGES_REQUESTED** [review]

**Review by dustymabe — APPROVED** [review]
LGTM

**Comment by travier** [comment]
Is this still a draft?

**Comment by dustymabe** [comment]
Ignition finally landed in all streams yesterday. I'll get the other PRs merged and then merge this too.

**Pull request merged**